app/controllers/catalog_controller.rb

```diff
@@ -294,7 +295,7 @@ class CatalogController < ApplicationController
     config.add_show_field 'recap_notes_display', label: 'RCP', helper_method: :recap_note
 
     # Numismatics fields
-    config.add_show_field 'issue_number_s', label: 'Issue', link_to_search: true, if: false
+    config.add_show_field 'issue_number_s', label: 'Issue', helper_method: :link_to_search_value
```

Comment:
keep the `if: false` so that the issue number can appear at the top of the page

Reply:
nice @tampakis , thanks you. i didn't know this blacklight magic.

Reply:
Orangelight magic, but magic nonetheless